staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -835,6 +835,18 @@ type CDISpec struct {
 	PriorityClass *CDIPriorityClass `json:"priorityClass,omitempty"`
 }
 
+type ComponentConfig struct {
```

Comment:
Any reason you wrapped this in a struct, instead of just adding the replicas to the CDISpec struct? Modifying the structure like that is going to affect a lot of people with existing installations.

Reply:
I just referred to the implementation of kubevirt, as well as the meaning of infra.
Breaking out of kubevirt will give me more freedom. In my opinion, we can add a replicas struct to the CDISpec struct.
```
type Replicas struct {
	deploymentReplicas *int32 `json:"deploymentReplicas,omitempty"`
	apiserverReplicas *int32 `json:"apiserverReplicas,omitempty"`
	uploadproxyReplicas *int32 `json:"uploadproxyReplicas,omitempty"`
}
```
In this way, replicas for each deployment can be set separately. Also I think the name of deploymentReplicas will make some people puzzle.